src/memo/memo.ml

```diff
@@ -1321,7 +1598,7 @@ let should_clear_caches =
     true
   | Some _ ->
     User_error.raise
-      [ Pp.text "Invalid value of DUNE_WATCHING_MODE_INCREMENTAL" ]
+      [ Pp.text "Unchanged value of DUNE_WATCHING_MODE_INCREMENTAL" ]
```

Comment:
Search&replace gone wrong :-)
(remember the `entd`)

Reply:
Oops :) This search & replace makes no sense to me, it's like replacing false with true.